web-crawler/src/main/java/org/iqu/webcrawler/entities/Event.java

```diff
@@ -0,0 +1,134 @@
+package org.iqu.webcrawler.entities;
+
```

Comment:
should be annotaned wil XMLRootElement and the toString method should be removed